CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 0.2.0-dev0
+
+### Enhancements
+
+* **Added migration for GitHub Source V2**
+  * **Updated the expected output for GitHub Source V2**
```

Comment:
Can we remove this line since it's related to the testing framework and not the code users use.

Reply:
Done.